src/middlewared/middlewared/plugins/apps/crud.py

```diff
@@ -93,9 +93,17 @@ def query(self, app, filters, options):
             return filter_list([], filters, options)
 
         extra = options.get('extra', {})
+        host_ip = extra.get('host_ip')
+        if not host_ip:
+            try:
+                if app.origin.is_tcp_ip_family:
+                    host_ip = app.origin.loc_addr
```

Comment:
Oh, below you handled IPv6.  Do we need to do so here too?

Reply:
@sonicaj do we need to put ipv6 address in brackets here for host_ip key in apps?